mods/Quack.md

```diff
@@ -17,48 +17,6 @@ Author: **easy bake oven#6781**
-| GetMousePos       | None, returns table                                                             | Gets the mouse's position on the screen.                                  |
-| GetMouseHeld      | None, returns bool                                                              | True if left mouse button is held, otherwise false.                       |
-| MessageBox        | (string message)                                                                | Shows a textbox on the user's screen. Limit of 3 at once.                 |
-| MessageBoxAsk     | (string message) returns bool                                                   | Shows a textbox on the user's screen with yes and no. True if yes.        |
-| MessageBoxIcon    | (string message, string icon)                                                   | Shows a textbox on the user's screen with specified icon. Icons at bottom.|
-| MessageBoxIconAsk | (string message, string icon) returns bool                                      | Same as MessageBoxAsk, but with an icon.                                  |
-| MessageBoxInput   | (string message, string default) returns string                                 | Asks user for text input with message.                                    |
-
-Some variables accessible to GetGooseProp and SetGooseProp:
-| Variable Name   | Purpose                                                   |
-|-----------------|-----------------------------------------------------------|
-| position        | Where the goose is on your screen. (table)                |
-| velocity        | The goose's speed. (table)                                |
-| direction       | The way the goose is facing, in degrees. (number)         |
-| targetDirection | The way the goose is trying to face, in degrees. (number) |
-| targetPos       | The position the goose is trying to get to. (table)       |
-| currentSpeed    | The current maximum speed of the goose. (number)          |
-
----
-
-## Color Table
-
-![Has a list of colors that can be used for the goose's color palette here https://docs.microsoft.com/en-us/dotnet/api/system.windows.media.colors?view=netframework-4.8](https://docs.microsoft.com/en-us/dotnet/media/art-color-table.png?view=netframework-4.8 "A  list of color that can be used")
-
-If you are unable to see the image you can click [here](https://docs.microsoft.com/en-us/dotnet/api/system.windows.media.colors?view=netframework-4.8)
-
-## Icons
-
-Error:
-![Error icon](https://docs.microsoft.com/en-us/dotnet/media/messagebox-error.png?view=netframework-4.8 "Error icon")
-
-Warning:
-![Warning icon](https://docs.microsoft.com/en-us/dotnet/media/messagebox-warning.png?view=netframework-4.8 "Warning icon")
-
-Information:
-![Information icon](https://docs.microsoft.com/en-us/dotnet/media/messagebox-information.png?view=netframework-4.8 "Information icon")
+## Documentation
```

Comment:
Could you add a source code section with a link to the repo itself too?

Reply:
ah yes. i shall preform such

Reply:
i have done so